pkg/policy/l4.go

```diff
@@ -134,6 +167,22 @@ func (l4 L4Filter) String() string {
 	return string(b)
 }
 
+func (l4 L4Filter) matchesLabels(labels labels.LabelArray) bool {
```

Comment:
@tgraf I wanted to double-check, if we have a policy like:

```
  [{                                                                            
    "endpointSelector": {"matchLabels":{"id.bar":""}},                          
    "ingress": [{                                                               
      "fromEndpoints": [                                                        
          {"matchLabels":{"reserved:host":""}},                                     
          {"matchLabels":{"id.foo":""}
      }]                                                                           
    }]                                                                    
  }]
```

Then this means that we should apply the policy at `id.bar` that the fromEndpoint must include either the label `reserved:host` or `id.foo`. It does not require both labels. Correct?

Reply:
Correct, fromEndpoints is a slice and each member is an allowed combination of labels. `matchLabels` can include multiple labels if more than label must match.